README.md

```diff
@@ -0,0 +1,22 @@
+# Soporte Remoto key handler
+
+This service is in charge of checking that the service keys exist and subscribing the communities that use it to the LibreNet6 network.
+Displays an endpoint POST /submit that receives the parameters generated by the federated-server on the FBW
+
+It also has two quite obvious commands:
+
+> npm run key:generate
+
+and
+
+>npm run key:delete [KEY]
+
+## Configuration
+
+This app uses dotenv to load the configuration, which can also be set by environment variables.
+
+#### Parameters used:
+
+- __DB__ database name
+- __PORT__ port where the server starts
+- __BOT_URL__ remote support bot url (in charge of adding the pubKey to librenet6)
```

Comment:
The documentation here helps a lot, but the name is still highly generic. Maybe __LNET6_BOT_URL__ will help identify quicker what it is about?

Reply:
Solved in https://github.com/LibreRouterOrg/soporteremoto-tiers/pull/1/commits/5ee45731302b6e5dfc1d5b6e8ccc50604c5b48e5